src/core/server/saved_objects/import/utils.ts

```diff
@@ -111,12 +130,22 @@ export const extractVegaSpecFromSavedObject = (savedObject: SavedObject) => {
   return undefined;
 };
 
-const isVegaVisualization = (savedObject: SavedObject) => {
+export const extractTimelineExpression = (savedObject: SavedObject) => {
+  if (confirmVisualizationType(savedObject, 'timelion')) {
```

Comment:
Nit: Logic should be the other way around. We should terminate early and return `undefined` if the visualization is not `timelion`